projects/plugins/boost/tests/e2e/specs/jetpack-boost/critical-css.test.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import { prerequisitesBuilder } from 'jetpack-e2e-tests/lib/env/prerequisites';
+
+/**
+ * Internal dependencies
+ */
+import JetpackBoostPage from '../../lib/pages/wp-admin/JetpackBoostPage';
+
+const moduleName = 'critical-css';
+let jetpackBoostPage;
+
+describe( 'Critical CSS module', () => {
+	beforeAll( async () => {
+		await prerequisitesBuilder().withLoggedIn( true ).withConnection( true ).build();
+	} );
+
+	beforeEach( async function () {
+		jetpackBoostPage = await JetpackBoostPage.visit( page );
+	} );
+
+	it( 'should be disabled by default', async () => {
```

Comment:
I really like how the spec looks now with the page functions. 


What I'm worried about though is the fact that all 3 tests depend on the default toggle state that is neither controlled nor checked. If the state is the non-default one, all 3 tests will fail.
I wonder if there's a way to reset the settings to their default state. This could be done as a test prerequisite.

Reply:
Ha yes, this is a good point. I have reworked it and introduce a new prerequisite file with commit 6be94f2f9c753cbbb6b08d550f536397c1ce70f5. Thought I have done some more refactoring after it such as removing individual module tests file.